### PR TITLE
Data object creation

### PR DESCRIPTION
**Pull request by takeller**
Created the classes to handle data object creation. Changed stat tracker to read the csv file and create arrays of game, game_teams, and team objects.

### CODE REVIEW

**Review by Lithnotep — APPROVED** [review]
Complete refactor of data acquisition.

File: test/game_teams_test.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    assert_equal 8, @game_team.pim
+    assert_equal 3, @game_team.power_play_opportunities
+    assert_equal 0, @game_team.power_play_goals
+    assert_equal 44.8, @game_team.face_off_win_percentage
+    assert_equal 17, @game_team.give_aways
+    assert_equal 7, @game_team.takeaways
+  end
+end
+
+# game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways
```
Comment (sagemlee) [inline test/game_teams_test.rb:36]: We can probably take out these comments.

File: test/game_test.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    assert_equal "6", @game.home_team_id
+    assert_equal 2, @game.away_goals
+    assert_equal 3, @game.home_goals
+    assert_equal "Toyota Stadium", @game.venue
+    assert_equal "/api/v1/venues/null", @game.venue_link
+  end
+end
+
+# game_id,season,type,date_time,away_team_id,home_team_id,away_goals,home_goals,venue,venue_link
+#
```
Comment (sagemlee) [inline test/game_test.rb:32]: We can probably take out these comments.

File: test/team_test.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    assert_equal "/api/v1/teams/1", @team.link
+  end
+end
+
+
+
+
+
+
+# team_id,franchiseId,teamName,abbreviation,Stadium,link
```
Comment (sagemlee) [inline test/team_test.rb:32]: Same with these

**Review by sagemlee — APPROVED** [review]

**Pull request merged**